src/test/schemamanager_test.cpp

```diff
@@ -1,99 +1,124 @@
 #include <gtest/gtest.h>
 
-#include <QSqlDatabase>
-#include <QTemporaryFile>
-#include <QtDebug>
-
 #include "test/mixxxtest.h"
-#include "library/trackcollection.h"
-#include "library/schemamanager.h"
+
+#include "database/mixxxdb.h"
+#include "database/schemamanager.h"
+
 #include "library/dao/settingsdao.h"
+
 #include "util/assert.h"
 
+
 class SchemaManagerTest : public MixxxTest {
   protected:
     SchemaManagerTest()
-            : m_dbFile("mixxxdb.sqlite") {
-        RELEASE_ASSERT(m_dbFile.open());
-        m_db = QSqlDatabase::addDatabase("QSQLITE");
-        m_db.setHostName("localhost");
-        m_db.setUserName("mixxx");
-        m_db.setPassword("mixxx");
-        m_db.setDatabaseName(m_dbFile.fileName());
-        RELEASE_ASSERT(m_db.open());
+            : m_mixxxDb(config()),
+              m_dbConnectionScope(m_mixxxDb.connectionPool()) {
+    }
+
+    const mixxx::DbConnectionPool::ThreadLocalScope& dbConnectionScope() const {
```

Comment:
It looks also misleading that we return a reference to a scoped connection. Can't we return the connection itself?